README.md

```diff
@@ -20,14 +20,12 @@ A live demo showcasing these features is available at http://craue.de/sf2playgro
 
 ## Get the bundle
 
-Let Composer download and install the bundle by running
+Let Composer download and install the bundle by running in a shell.
 
 ```sh
-php composer.phar require craue/formflow-bundle:~2.0
+php composer.phar require craue/formflow-bundle:dev-master
 ```
 
-in a shell.
```

Comment:
For me it is a sentence in its original form. The code could be inlined to make it look better, but it's like that in all my bundles currently and I'd like to keep it consistent.